datadog/tests/resource_datadog_metric_tag_configuration_test.go

```diff
@@ -47,6 +51,7 @@ func testAccCheckDatadogMetricTagConfigurationIncludePercentilesError(uniq strin
 			metric_name = "%s"
 			metric_type = "%s"
 			tags = ["sport"]
+            exclude_tags_mode = false
```

Comment:
nit: can we fix the formatting so its in line with the other params (same below)